package.json

```diff
@@ -87,10 +87,10 @@
     "@preconstruct/cli": "^2.1.0",
     "@types/node": "^16.0.1",
     "@typescript-eslint/eslint-plugin": "^4.23.0",
-    "@typescript-eslint/parser": "^4.23.0",
+    "@typescript-eslint/parser": "^5.37.0",
     "concurrently": "6.2.0",
     "esbuild": "^0.14.48",
-    "eslint": "^7.26.0",
+    "eslint": "^8.0.0",
```

Comment:
We have the config in `.eslintrc.js` commented out so updating these doesn't affect anything